backend/router/email_crud.py

```diff
@@ -32,7 +58,16 @@ def password_reset(req: EmailResetRequest, db: Session = Depends(get_db)):
         raise HTTPException(status_code=404, detail="User not found")
 
     reset_token = str(uuid.uuid4())
-    reset_link = f"http://localhost:5173/reset-password?token={reset_token}"
+    password_reset = PasswordReset(
+        user_id=user.id,
+        token=reset_token,
+        created_at=datetime.now(),
+    )
+    db.add(password_reset)
+    db.commit()
+    FRONTEND_URL = os.getenv("FRONTEND_URL")
+    reset_link = f"{FRONTEND_URL}/reset-password?token={reset_token}"
```

Comment:
nie, ja som chcel aby to vrátilo iba ten token a frontend by potom vytvoril odkaz na `/reset-password?token=<ten token>` (cez `<a href="/..."`), presne z toho dôvodu aby sme obišli to že backend si musí pamätať aj adresu frontendu atď